### ISSUE TITLE
`#![feature]` may not be used on the stable release channel

### ISSUE THREAD

**Issue by adan-ea** [sparten11740/bevy_health_bar3d#32]
Hi there, this plugin seem really interesting to use but i seem to find myself in a pickle. 
When I download it and try to run my code i get this error 
```log
   Compiling bevy_health_bar3d v2.0.1
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> C:\Users\adan-ea\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bevy_health_bar3d-2.0.1\src\lib.rs:1:1
  |
1 | #![feature(exhaustive_patterns)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `bevy_health_bar3d` (lib) due to 1 previous error
```

i don't get how to fix this, i've tried multiple versions of this crate with no success. 
PS: I'm still on bevy12.0 because one of the other plugin isn't up to date with 0.13 :')

here is my cargo.toml for a bit more context: 
```toml
[package]
name = "astro-odyssey"
version = "0.1.0"
edition = "2021"

[dependencies]
# Remove the features before production
bevy = { version = "0.12", features = ["dynamic_linking"] }
bevy-inspector-egui = "0.22"
bevy_ecs_tilemap = "0.12.0"
bevy_health_bar3d = "2.0.0"
bevy_pancam = "0.10"
noise = "0.9.0"
rand = "0.8.5"

[workspace]
resolver = "2" # Important! wgpu/Bevy needs this!

# Enable a small amount of optimization in debug mode
[profile.dev]
opt-level = 1

# Enable high optimizations for dependencies (incl. Bevy), but not for our code:
[profile.dev.package."*"]
opt-level = 3
```

rustc 1.76.0 (07dca489a 2024-02-04)

**Comment by sparten11740** [comment]
hi, thanks for opening the issue. The `exaustive_patterns` have actually become obsolete with a previous PR. I removed it in https://github.com/sparten11740/bevy_health_bar3d/pull/33. The reason why rustc complains in your case is that you're on rust stable and not nighly

**Comment by sparten11740** [comment]
I backported the fix as v3.0.0-hotfix.0 to the version that works with bevy 0.12: https://crates.io/crates/bevy_health_bar3d/3.0.0-hotfix.0
